src/xbot/common/logging/SilentRobotAssertionManager.java

```diff
@@ -0,0 +1,15 @@
+package xbot.common.logging;
+
+public class SilentRobotAssertionManager extends RobotAssertionManager {
+
+    @Override
+    protected void handlePlatformException(RuntimeException e) {
+        // Don't do anything: we don't need to throw
```

Comment:
Log a warning! We don't want the robot to die, but we should probably know about this.